.github/workflows/release.yml

```
@@ -141,7 +141,7 @@ jobs:
           CMD="go version | awk '{print \$3}' | sed -e 's/^go//g'"
           go_version=$(docker compose -f docker-compose.yml -f dev.docker-compose.yml run server sh -c "${CMD}")
           echo "Go version:" "${go_version}"
-          echo "::set-output name=go_version::${go_version}"
+          echo "go_version=${go_version}" >> $GITHUB_OUTPUT
```

https://github.com/dev-hato/hato-atama/actions/runs/3243509565/jobs/5318324765

```
.github/workflows/release.yml:140:9: shellcheck reported issue in this script: SC2086:info:4:36: Double quote to prevent globbing and word splitting [shellcheck]
    |
140 |         run: |
    |         ^~~~
```

他もおそらく同様。

---

```suggestion
          echo "go_version=${go_version}" >> "${GITHUB_OUTPUT}"
```

こんな感じにしてあげればおそらく良さそう。

---

なるほどきびしい